libs/frontend/feature-render-builder/src/components/ProseMirrorEditor.ts

```diff
@@ -203,7 +203,7 @@ export const ProseMirrorEditor = defineComponent({
 
     return () => {
       return h('div', {
-        id: containerId,
+        id: containerId.value,
```

Comment:
I accidentally saw an element with id `[object Object]` in the inspector, and this seems to fix it.

Reply:
Good catch 😬